oauth2/src/test/groovy/com/okta/spring/boot/oauth/AutoConfigConditionalTest.groovy

```diff
@@ -355,7 +354,7 @@ class AutoConfigConditionalTest implements HttpMock {
             "spring.security.oauth2.client.provider.okta.issuerUri=${mockBaseUrl()}oauth2/custom-as", // work around to not validate the https url
             "okta.oauth2.client-id=test-client-id",
             "okta.oauth2.client-secret=test-client-secret",
-            "okta.oauth2.postLogoutRedirectUri=http://logout.example.com")
+            "okta.oauth2.postLogoutRedirectUri=/logout/callback")
```

Comment:
Add new tests for the new functionality, updating existing tests to change the test expectation removes the ability to catch regressions, for the previous functionality.

This particular change could probably be done with a simple UT instead of spinning up a spring context, but I'll leave that one up to you.

Reply:
done